assets/src/edit-story/app/rightClickMenu/constants.js

```diff
@@ -48,6 +54,46 @@ export const RIGHT_CLICK_MENU_SHORTCUT_LABELS = {
     'web-stories'
   ),
   DELETE: _x('Delete', 'The keyboard key "Delete"', 'web-stories'),
+  COMMAND_OPEN_BRACKET: _x(
```

Comment:
What are these used for? `aria-label`?

Could we do this differently. I worry that translators won't be translating these correctly... Plus there's lots of repetition.

Could we perhaps reuse `SPECIAL_KEYS` from the keyboard shortcuts menu somehow?

That said, is using `aria-label` the best way to announce the keyboard shortcuts to users? Are there other options?

cc @barklund for thoughts

Reply:
These are used as the [label text for the context menu](https://github.com/google/web-stories-wp/blob/main/packages/design-system/src/components/contextMenu/menuItem.js#L78-L116) - #7911 

Yeah there's a lot of repetition 😕. I did it this way because the translate function arguments must be string literals. This is tricky since we want to have dynamic keys here based on operating system.

>Could we perhaps reuse `SPECIAL_KEYS` from the keyboard shortcuts menu somehow?

Chatted with max and I'm updating these to use the `kbd` element. 
![image](https://user-images.githubusercontent.com/22185279/127914197-211939f2-5fc6-4d71-9eef-71c64781d26e.png)

The context menu also uses [`kbd` for the `Shortcut`](https://github.com/google/web-stories-wp/blob/main/packages/design-system/src/components/contextMenu/menuItem.js#L87-L116).

Reply:
Just updated these! Let me know what you think 😄

Reply:
@swissspidy Thoughts on this? It's semi-blocking so we'd like to get the rest of this in. We can always come back to this too since the right click menu is a work in progress! it's not user facing

Reply:
Going to merge this but let's keep chatting about the above! We can come back and fix this very easily.